### PR TITLE
Project details page

### PR DESCRIPTION
**Pull request by Prevole**

### CODE REVIEW

File: app/assets/javascripts/modules/routes.js [hunk 1]
```diff
@@ -150,6 +155,16 @@ angular.module('probedock.routes', [ 'ui.router' ])
         url: '/:id'
       })
 
+      .state('org.project', {
+        abstract: true,
+        template: '<div ui-view />'
+      })
+
+      .state('org.project.show', {
```
Comment (AlphaHydrae) [inline app/assets/javascripts/modules/routes.js]: For consistency with `org.users.show`, use `org.projects.show` (and thus remove the duplicate previous abstract state).

File: app/views/templates/project.html.slim [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  .page-header
+    h1 {{ project | projectName }}
+  .row.overview
+    .col-lg-3.col-md-4.col-xs-12
+      .summary.panel.panel-default ng-show='project.id'
+        .panel-heading.clearfix
+          .label.label-info.pull-right tooltip='Project ID' {{ project.id }}
+          h3.panel-title Details
+        .panel-body
+          ul
```
Comment (AlphaHydrae) [inline app/views/templates/project.html.slim]: For consistency with other details pages (project list, user details), use a definition list here. The description may be displayed as a `p.lead` element to make it stand out, like in the project list (see `project-panel.html.slim`).

File: app/views/templates/project.html.slim [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            li ng-show='project.description' {{ project.description }}
+            li {{ project.testsCount | number }} tests in the project
+            li {{ project.reportsCount | number }} test runs
+            li Created on {{ project.createdAt | amDateFormat:'llll' }}
```
Comment (AlphaHydrae) [inline app/views/templates/project.html.slim]: Prefer the more human-friendly and explanatory terms "Creation Date" (used in project list) and "Last Update" for the definition terms.

File: features/reports_project_filter.feature [hunk 4]
```diff
@@ -0,0 +1,182 @@
+@api @reports
+Feature: Reports
+
+  After publishing test results to Probe Dock for different projects within an organization, users of that
+  organization should be able to access a list of reports for a specific project.
+
+
+
+  Scenario: An organization member should be able to get a list of reports for a specific project and a private organization by ID.
```
Comment (AlphaHydrae) [inline features/reports_project_filter.feature]: Please add the `@search` tag to scenarios pertaining to list filters.

File: features/reports_project_filter.feature [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+          "createdAt": "@iso8601",
+          "organizationId": "@idOf: Rebel Alliance"
+        }
+      ]
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: A member in a different organization should be able to get a list of reports for a specific project from a public organization by ID.
```
Comment (AlphaHydrae) [inline features/reports_project_filter.feature]: I would remove the "by ID" at the end of the description of all scenarios in this file. It is unclear here what ID we are talking about (organization or project), and not necessary to understanding the test.

File: app/views/templates/project.html.slim [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.container-fluid
+  .page-header
+    h1 {{ project | projectName }}
+  .row.overview
+    .col-lg-3.col-md-4.col-xs-12
+      .summary.panel.panel-default ng-show='project.id'
+        .panel-heading.clearfix
+          .label.label-info.pull-right tooltip='Project ID' {{ project.id }}
```
Comment (AlphaHydrae) [inline app/views/templates/project.html.slim]: Add the `.apiId` class to this label so that the ID is displayed with a Monospace font.

File: app/assets/javascripts/modules/routes.js (hunk 1 above)
Comment (Prevole) [inline app/assets/javascripts/modules/routes.js]: Done

File: app/views/templates/project.html.slim (hunk 2 above)
Comment (Prevole) [inline app/views/templates/project.html.slim]: Done and visually reviewed by you :)

File: app/views/templates/project.html.slim (hunk 3 above)
Comment (Prevole) [inline app/views/templates/project.html.slim]: Same there

File: features/reports_project_filter.feature (hunk 4 above)
Comment (Prevole) [inline features/reports_project_filter.feature]: Only added on scenarios that are not related to authorization, correct ?

File: features/reports_project_filter.feature (hunk 5 above)
Comment (Prevole) [inline features/reports_project_filter.feature]: Done

File: app/views/templates/project.html.slim (hunk 6 above)
Comment (Prevole) [inline app/views/templates/project.html.slim]: Done

File: features/reports_project_filter.feature (hunk 4 above)
Comment (AlphaHydrae) [inline features/reports_project_filter.feature]: Correct.
